src/server/snap.rs

```diff
@@ -185,6 +188,20 @@ pub fn send_snap(
             Ok(_) => {
                 fail_point!("snapshot_delete_after_send");
                 mgr.delete_snapshot(&key, &chunks.snap, true);
+                let cost = UnixSecs::now().into_inner() - snapshot_tracker.start;
+                // it should ignore if the duration of snapshot is less than 1s to decrease the
+                // grpc data size.
+                if cost >= 1 {
+                    let mut stat = SnapshotStat::default();
+                    stat.set_region_id(snapshot_tracker.region_id);
```

Comment:
it's better to unify use `snapshot_tracker` to trace the all stage, and then `mgr.collect_stat(snapshot_tracker)`

Reply:
yes，I will remove `snapshot_tracker` and only use `SnapshotStats`